distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Link } from 'react-router';
+import axios from 'axios';
+
+
```

Comment:
nitpick: 1 linebreak is fine. Was trying to separate lib imports from project ones.